roles/DCOS.bootstrap/tasks/main.yml

```diff
@@ -36,9 +36,18 @@
 
 # PRE DOWNLOAD vvv
 
+- name: "Install EPEL repositories for required packages"
```

Comment:
Is this needed for Bootstrap? Its already here: https://github.com/dcos/dcos-ansible/blob/master/roles/DCOS.requirements/tasks/main.yml#L80-L87

Reply:
oh you are right, only my fault, will remove